### PR TITLE
STOR-1804: update ClusterCSIDriver CRDs

### PR DESCRIPTION
**Pull request by RomanBednar**

### CODE REVIEW

File: tools.go
Comment (jsafrane) [inline tools.go:1]: Put new dummy deps to pkg/dependencymagnet/dependencymagnet.go

Comment (RomanBednar) [reply tools.go:1]: Corrected.

**Comment by Phaow** [comment]
/label qe-approved
Pre verify passed with 4.16.0-0.test-2024-04-08-003725-ci-ln-sv9g1z2-latest (build 4.16.0-0.ci,openshift/api#1783,openshift/vmware-vsphere-csi-driver-operator#226,openshift/cluster-storage-operator#464)

**Pull request merged**